### ISSUE TITLE
I cannot change password, when I reset password.

### ISSUE THREAD

**Issue by prugel** [npm/newww#673]
https://www.npmjs.com/password

login ok, but When I click change password button, 

display below code.

```
{
statusCode: 400,
error: "Bad Request",
message: "/password"
}
```

**Comment by chadxz** [comment]
happening for me currently as well.

**Comment by rockbot** [comment]
@prugel, @chadxz - what are your npm usernames? (If you would prefer to tell us via email, please reach out at support@npmjs.com)

**Comment by chadxz** [comment]
I emailed you my username. Tried resetting password again today and got same error.

**Comment by prugel** [comment]
I try firefox, It works.
but chrome does not.

**Comment by lmorchard** [comment]
+1 to this error for me, too - username: lmorchard

**Comment by kennethkalmer** [comment]
Same happened to me, but when I used my username (instead of email) in combination with the temporary password, it finally worked and I could change me password...

hth

**Comment by rockbot** [comment]
@prugel: any chance it's due to this issue? https://github.com/npm/newww/issues/694

@chadxz, @lmorchard - has it been fixed for you?

**Comment by lmorchard** [comment]
Hmm, I am indeed a LastPass user! And it had saved a form field named "honey", which when deleted seems to make things happier.

**Comment by chadxz** [comment]
@rockbot following @lmorchard's instructions, I am able to happily use Lastpass to manage my npmjs.com website login credentials.

first I had to edit the saved password and expand "Advanced Settings", then click "edit form fields"
![screen shot 2015-03-16 at 3 44 14 pm](https://cloud.githubusercontent.com/assets/309219/6676115/83af914a-cbf3-11e4-8cee-7995efc4720e.png)

Then I had to find the field labelled "honey" and just click the '-' button next to it to remove it from the autofill.
![screen shot 2015-03-16 at 3 52 01 pm](https://cloud.githubusercontent.com/assets/309219/6676283/b4bef536-cbf4-11e4-8243-50cc7e583e9d.png)

after that things worked swimmingly. Thanks @lmorchard and @rockbot !
